app/controllers/patients_controller.rb

```diff
@@ -57,6 +57,11 @@ def destroy
     redirect_to patients_path
   end
 
+  def search_patient
+    patients = Patient.whose_name_starts_with(params[:name])
```

Comment:
The search no necessarily needs to be based on person first name
What If I only remember person last name?

Reply:
Does not make sense for me, because we just have an attribute called `name` that contains the full name. Idk if the user will search by another criteria

Reply:
So the problem is your scope `whose_name_starts_with`
It's name indicates that it will look for people whose names **start** with a particular argument; say 'Dieg'

That is **not true**. You're using **full text search**
If you want to get people whose name start with **something**, you can achieve it with a simple `LIKE "Diet%"` SQL statement...but this is not the case

Reply:
Ok, so what about `whose_fullname_contains`?